src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -37,6 +37,17 @@ def load_arguments(self, _):
     with self.argument_context('spring-cloud update') as c:
         c.argument('sku', type=str, validator=validate_sku, help='Name of SKU, the value is "Basic" or "Standard"')
 
+    for scope in ['spring-cloud create', 'spring-cloud update']:
+        with self.argument_context(scope) as c:
+            c.argument('app_insights_key',
+                       help="Instrumentation key of the existing Application Insights to be added for the distributed tracing")
+            c.argument('app_insights',
+                       help="Name of the existing Application Insights in the same Resource Group. Or Resource ID of the existing Application Insights in a different Resource Group.")
+            c.argument('disable_distributed_tracing',
+                       arg_type=get_three_state_flag(),
+                       help="Disable distributed tracing, if not disabled and no existing Application Insights specified with --app-insights-key or --app-insights, will create a new Application Insights instance in the same resource group.")
```

Comment:
Another reason to use `action='store_true'` is that this is not a property of resource, but a switch to control the usage of application insights. See the last example of [Registering Flags](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags).

Reply:
Have fixed, thanks a lot